book/10_middleware.md

```diff
@@ -15,13 +15,52 @@ class MyMiddleware
   end
   def call(env)
     status, headers, body = @app.call(env)
-    [status, headers, body]
+    # Do something to status, headers, and body here...
```

Comment:
Is this a TODO and should it stay here on merge?

Reply:
Yes I think it's a suggest for how to use the middleware.

Reply:
Gotcha, thanks!